src/util/util.cpp

```diff
@@ -136,7 +136,7 @@ std::string to_lower_case(const std::string& s)
 std::string print_char(char c)
 {
 	char buf[16];
-	if (c < 32 && c >= 0)
+	if (c < 32 && c > 0)
```

Comment:
why exclude 0?

Reply:
It fixes this warning
```
src/util/util.cpp:139:25: warning: comparison is always true due to limited range of data type [-Wtype-limits]
  139 |         if (c < 32 && c >= 0)
      |                       ~~^~~~
```

Reply:
If we really want `c >= 0` then it would be best to just remove that clause.

```suggestion
	if (c < 32)
```